app.js

```diff
@@ -0,0 +1,18 @@
+const form = document.querySelector('form');
+const input = document.querySelector('input');
+
+
+addTask = (e) => {
+    e.preventDefault();
+    const titleTask =  input.value;
+    const ul = document.createElement('ul');
+    const li = document.createElement('li');
+    ul.appendChild(li);
+    document.body.appendChild(ul);
```

Comment:
Why do you use `ul` element as single item? You should use `li` element for that